Documentation/ReleaseNotes.md

```diff
@@ -0,0 +1,285 @@
+# Microsoft Mixed Reality Toolkit Release Notes
+
+- [Version 2.0.0](#version-200)
+
+## Version 2.0.0
+
+- [Upgrading projects](#upgrading-projects-to-200)
+- [What's new](#whats-new-in-200)
+- [Known issues](#known-issues-in-200)
+
+This release of the Microsoft Mixed Reality Toolkit 2.0.0 supports the following devices and platforms.
+
+- Microsoft HoloLens 2
+- Microsoft HoloLens (1st gen)
+- Windows Mixed Reality Immersive headsets
+- OpenVR
+
```

Comment:
Some people also use ARKit, ARCore. Should we mention that MRTK will compile and run for these other platforms, however those platforms have some issues?

Reply:
I wasnt planning on it, since we do not formally support using MRTK systems and features on those devices